package.json

```
@@ -41,7 +41,7 @@
     "electron-updater": "^4.3.9",
     "emittery": "^0.10.0",
     "googleapis": "^87.0.0",
-    "israeli-bank-scrapers-core": "^1.5.0",
+    "israeli-bank-scrapers-core": "^1.5.3",
```

```suggestion
    "israeli-bank-scrapers-core": "1.5.3",
```

Please use the exact version.
We are trusting this library, but we should be careful, to not be affected if it will be hijacked or something.